test/helpers/framework.go

```diff
@@ -63,24 +63,21 @@ type LogStore interface {
 	HasAuditLogs(timeToWait time.Duration) (bool, error)
 
 	GrepLogs(expr string, timeToWait time.Duration) (string, error)
-
-	ClusterLocalEndpoint() string
 }
 
 type E2ETestFramework struct {
 	RestConfig     *rest.Config
 	KubeClient     *kubernetes.Clientset
 	ClusterLogging *cl.ClusterLogging
 	CleanupFns     []func() error
-	LogStores      map[string]LogStore
+	LogStore       LogStore
```

Comment:
Please elaborate why switching back to a single store? The `map[string]LogStore` addressed writing tests with multiple log stores as in `test/e2e/logforwarding/multiple_outputs/forward_to_multiple_outputs_test.go`.